### ISSUE TITLE
[Bug : Tour Book view] Duplicate Tour do not work anymore

### ISSUE THREAD

**Issue by GurVJ** [mytourbook/mytourbook#1041]
### What happened ?

The 23.30 release notes is stating taht the "The action Duplicate Tour in the "Tour Book" context menu is now enabled for all tours, not only for manual tours as it was before." but actually  i cannot duplicate manual tours anymore ...
I am trying to do it using the contex menu "Duplicate Tour" in the "Categorized by Year/Month" tour book view and nothing happens (except that I've got the warning message concerning the experimental feature).

### Version

22.3.0

### System

Windows

**Comment by wolfgang-ch** [comment]
When a tour contains tags, then duplicating this tour is causing an exception, this is independent if the tour is manual or not.

Duplicating a tour do not check any more if it was created manually or is from a device.

```

!ENTRY net.tourbook.common 1 0 2023-03-26 07:55:36.853
!MESSAGE 07:55:36.804'647 [System Property - TourbookDevice] - "createRandomTourId" is enabled -> Every imported tour has a unique tour ID
07:58:32,369 ERROR LazyInitializationException:42 - failed to lazily initialize a collection of role: net.tourbook.data.TourTag.tourData, no session or session was closed
org.hibernate.LazyInitializationException: failed to lazily initialize a collection of role: net.tourbook.data.TourTag.tourData, no session or session was closed
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationException(AbstractPersistentCollection.java:380)
	at org.hibernate.collection.AbstractPersistentCollection.throwLazyInitializationExceptionIfNotConnected(AbstractPersistentCollection.java:372)
	at org.hibernate.collection.AbstractPersistentCollection.initialize(AbstractPersistentCollection.java:365)
	at org.hibernate.collection.PersistentSet.add(PersistentSet.java:212)
	at net.tourbook.data.TourTag.setupDeepClone(TourTag.java:326)
	at net.tourbook.data.TourData.createDeepCopy(TourData.java:5714)
	at net.tourbook.ui.views.tourDataEditor.TourDataEditorView.actionCreateTour(TourDataEditorView.java:2062)
	at net.tourbook.ui.action.ActionDuplicateTour.run(ActionDuplicateTour.java:67)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:474)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:580)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:414)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4251)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4068)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3645)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:551)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:156)
	at net.tourbook.application.Application.start(Application.java:35)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1440)

```

**Comment by wolfgang-ch** [comment]
Fixed in 23.3.2
